src/github.com/matrix-org/dendrite/clientapi/jsonerror/jsonerror.go

```diff
@@ -64,6 +64,12 @@ func UnknownToken(msg string) *MatrixError {
 	return &MatrixError{"M_UNKNOWN_TOKEN", msg}
 }
 
+// InvalidSync is an error when the client tries to hit /sync with an invalid
+// ?since= parameter.
+func InvalidSync(msg string) *MatrixError {
+	return &MatrixError{"M_BAD_SYNC", msg}
```

Comment:
We can always change it to something else some other time I guess. Done. https://github.com/matrix-org/dendrite/pull/57/commits/13f8237c5429e73626a04991e6c7a6b0b6b9e26f